README.md

```diff
@@ -138,6 +138,28 @@ Class: `o-table--vertical-lines`, Mixin : `oTableVerticalLines`
 
 Thin lines will be rendered to the left and right of each `td` element giving the appearance of lines between columns.
 
+## Responsive
+
+In the fifth version of `o-table`, there are responsive options that can be used for displaying data in a table. 
+
+### Flat
+
+Class: `o-table--responsive-flat`, Mixin: `oTableResponsiveFlat`
+
+Using the Responsive Flat version will render the table to change at narrow viewpoints into a row-based table with each row having a duplicate of the table headers on the left side. This uses JavaScript (`responsive-flat.js`) to inject the headers into each row.
```

Comment:
you don't need to mention the JS file, but do mention that it doesn´t work in Core